prusti-tests/tests/verify/pass/arrays/slice-split.rs

```diff
@@ -0,0 +1,19 @@
+use prusti_contracts::*;
+
+fn main() {
+    let a = [0; 13];
+    let (s0, s1) = split_at(&a, 7);
+
+    assert!(s0.len() == 7);
+    assert!(s1.len() == 6);
```

Comment:
Given the postconditions, you should also be able to prove assertions about concrete values (e.g. `assert!(s0[2] == 0);`), is that right? If so, please add an assertion like that.